lib/bfd.c

```diff
@@ -94,22 +94,33 @@ int bfd_validate_param(struct vty *vty, const char *dm_str, const char *rx_str,
  * bfd_set_param - Set the configured BFD paramter values
  */
 void bfd_set_param(struct bfd_info **bfd_info, uint32_t min_rx, uint32_t min_tx,
-		   uint8_t detect_mult, int defaults, int *command)
+		   uint8_t detect_mult, char *profile, int defaults,
+		   int *command)
 {
+	char profile_sanitized[BFD_PROFILE_NAME_LEN];
+
+	if (profile)
```

Comment:
This is dead ugly, will find a better solution for this tomorrow. Hints welcome.